.changelog/1548.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:feature
+core: Add support for input variables
```

Comment:
Just a note that this is surely a candidate for **bold feature** (that we don't support automating yet). So no action you have to do @krantzinator but just for us to know when we cut a release and build the changelog.

Reply:
10-4